eventhubs-streamanalytics-eventhubs/tools/README.md

```diff
@@ -1,11 +1,41 @@
-# Testing End-To-End Latency
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure eventhubs 
+extensions:
+- platforms: dotnet
+---
 
-Get the Event Hub connection string created and displayed during the script execution. If you cannot find it, just grab the execution string for the `Listen` Shared Access Policy for the output eventhub.
+# Getting started on processing received message from eventhub #
```

Comment:
Please change to `Event Hubs` to reflect the name of the product.

Reply:
resolved